README.md

```diff
@@ -137,6 +137,7 @@ preview.show(fileId, accessToken, {
 | File Option | Description |
 | --- | --- |
 | fileVersionId | File version ID to preview. This must be a valid non-current file version ID. Use [Get Versions](https://developer.box.com/reference#view-versions-of-a-file) to fetch a list of file versions |
+| startAt | The unit and value of where to start the preview at. Current supported units are 'seconds' for media and 'pages' for documents. |
```

Comment:
`Object with unit and value properties indicating where to start the preview at. ...`